File: desktop/src/com/llgames/ia/desktop/DesktopLauncher.java
package com.llgames.ia.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.llgames.ia.IAGame;

public class DesktopLauncher {
    public static void main (String[] arg) {
        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.width = 640;
        config.height = 360;
        new LwjglApplication(new IAGame(), config);
    }
}

File: desktop/src/com/llgames/ia/desktop/DesktopLauncher.kt
@file:JvmName("DesktopLauncher")

package com.llgames.ia.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.llgames.ia.SelectIA

fun main(arg: Array<String>) {
    val config = LwjglApplicationConfiguration().apply {
        title = "Battle Desktop"
        width = 640
        height = 360
    }
    LwjglApplication(SelectIA(), config)
}
